public/components/overview/mitre/mitre.tsx

```diff
@@ -120,14 +123,17 @@ export class Mitre extends Component {
       result && result.forEach(item => {
           const {id, phase_name} = item;
           phase_name.forEach( (tactic) => {
-            if(!tacticsObject[tactic]){ 
+            if(!tacticsObject[tactic]){
               tacticsObject[tactic] = [];
             }
             tacticsObject[tactic].push(id);
           })
         });
       this._isMount && this.setState({tacticsObject, isLoading: false});
-    }catch(err){
+    } catch(err) {
+      if (err.match('3013 - Permission denied')) {
+        this.setState({isLoading: false, notPermissions: true});
+      }
```

Comment:
Here you are managing the error with the condition. But, the loading status continues if the error does not match the condition because it is not set to `false`.

Reply:
Ok we can set `isLoading` false without condition 👍🏼